bee-message/tests/dkg.rs

```diff
@@ -0,0 +1,143 @@
+// Copyright 2021 IOTA Stiftung
```

Comment:
For consistency, can you end the payload test file names with `_payload` please ?